AlgebraSurvey.tex

```diff
@@ -41,12 +41,12 @@ \section{Proof Assistant Systems}
 discussed in the previous chapter. 
 
 \subsection{Coq}
-Coq \cite{Paulin-Mohring2012} is a theorem-proving system that is written in the
 OCaml programming language. It is designed to assist in the formalization of
-mathematical theorems by using constructive and higher-order logic.  It was
-first released in 1989 and is one of the most widely used proof assistant
-systems to define mathematical definitions, and theory and to write proofs. The
-Coq specification language is called Gallina. Users can write functions and
+Coq \cite{Paulin-Mohring2012} is a theorem-proving system that is written in the
```

Comment:
this now makes no sense as the 2nd line has been flipped with the first! You should be more careful

Reply:
Ah, I select sentences while reading. When I move the mouse (by mistake), vscode will shift the selected part. This change was not intentional.